Example/URBNJSONDecodable-Example.xcodeproj/project.pbxproj

```diff
@@ -494,7 +505,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.urbn.URBNJSONDecodable-Example";
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
I commented on the other ticket, but we might want to look into this at some point